cli/parser/parser.go

```diff
@@ -246,6 +249,18 @@ type Option struct {
 // used to parse the flag schema for the particular topic.
 type BazelHelpFunc func(topic string) (string, error)
 
+func ParseBazelHelpProto(help string) *OptionSet {
+	b, err := base64.StdEncoding.DecodeString(help)
+	if err != nil {
+		return nil
+	}
+	m := &bfpb.FlagCollection{}
+	if err := proto.Unmarshal(b, m); err != nil {
+		return nil
+	}
+	return nil
+}
+
```

Comment:
I think this is unused?

```suggestion
```

Reply:
Oh, yeah, that was from when I was just trying to get it to import correctly; good catch!